src/helpers/objects.processor.js

```diff
@@ -39,6 +39,17 @@ function injectInto(locator, injection) {
     return locator + '[' + injection;
 }
 
+/**
```

Comment:
This function should be moved to the utils (see _r for example). Only Object processing methods should be placed here